tests/Unit/NumericalAlgorithms/Integration/Test_GslQuad.cpp

```diff
@@ -0,0 +1,116 @@
+// Distributed under the MIT License.
```

Comment:
Please name this file `Test_GslQuadAdaptive.cpp` so it's consistent with the file it tests.